libs/ast_generic/AST_generic.ml

```diff
@@ -484,7 +483,6 @@ class virtual ['self] map_parent =
     method visit_resolved_name _env x = x
     method visit_tok _env x = x
     method visit_node_id_t _env x = x
-    method visit_string_set_t _env x = x
```

Comment:
@nmote FYI I had a very cryptic error message by ocamlc with an error when instantiating a visitor.
Once I removed s_backrefs below, this string set was not used, so the visit_string_set_t suddently contains
some free variables and got an error like
"lazy_t contains free variables, impossible to instantiate".
I got lucky I found it was because of those visit_string_set_t methods ...

Reply:
Maybe all of these should have type annotations?